101-vmms-with-public-ip-prefix/metadata.json

```diff
@@ -0,0 +1,9 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/...",
```

Comment:
this is not a valid $schema property (see other samples)

Reply:
Changed to valid $schema property as found in other templates